src/state/reducers/dataReducer.js

```diff
@@ -1,16 +1,67 @@
-import { GET_DATA } from '../constants';
+import { combineReducers } from 'redux';
+import {
+  GET_DATA,
+  INPUT_CHANGE,
+  FILTER_SEARCH,
+  FILTER_CATEGORIES,
+  RESET_FILTER,
+} from '../constants';
 
-const initialState = {
-  asylum: [],
+// States
+const initialAsylumData = [];
+
+const initialFilteredStates = {
+  categories: [
+    'asylumOffice',
+    'citizenship',
+    'raceOrEthnicity',
+    'caseOutcome',
+    'completion',
+    'currentDate',
+  ],
+  count: 0,
+  data: [],
+};
+
+const initialFormState = {
+  searchTerm: '',
+  category: '',
 };
 
-const dataReducer = (state = initialState, action) => {
+// Reducers
+function asylumReducer(state = initialAsylumData, action) {
   switch (action.type) {
     case GET_DATA:
       return action.payload;
     default:
       return state;
   }
-};
+}
+
+function filteredReducer(state = initialFilteredStates, action) {
+  switch (action.type) {
```

Comment:
Consider wrapping the resulting operation from each switch case in squiggly brackets in order to create block scope and prevent unintended modification of variable values which might result from too expansive a scope(presently, the scope for each statement is the entire switch statement)

Reply:
I went ahead and updated the dataReducer switch cases based on our recommendation.